services/worker-manager/src/util.js

```diff
@@ -23,3 +24,24 @@ const joinWorkerPoolId = (provisionerId, workerType) => {
 };
 exports.joinWorkerPoolId = joinWorkerPoolId;
 exports.MAX_MODIFY_ATTEMPTS = 5;
+
+const createCredentials = (worker, expires, cfg) => {
```

Comment:
The comment above this in `registerWorker` explains the importance of using only values that we've generated/stored for this interpolation. I think it would be worth mentioning that here too for future uses